py/selenium/webdriver/common/action_chains.py

```diff
@@ -79,7 +79,7 @@ def reset_actions(self):
         for device in self.w3c_actions.devices:
             device.clear_actions()
 
-    def click(self, on_element=None):
+    def click(self, on_element=None, device=None):
```

Comment:
@titusfortner Here's a tricky case which I just realised.

I have added `device` to click() and in lines 93 & 94 that device is passed to `key_action.pause()` signifying that device is `KeyInput`. This works with `send_keys_to_element()` (line 290) which calls `click()`. The tests pass.

Now what if there's a case where the user does 
```
mouse = PointerInput(interaction.POINTER_MOUSE, "test mouse")
ActionChains(driver).click(device=mouse).perform()
```
here the click() receives a `PointerInput` device and logically it should pass this device to `pointer_action.click()` (line 92) and **NOT** in `key_action.pause()` (lines 93 and 94)

Will the user ever do `ActionChains(driver).click(device=mouse).perform()` ?

Because the below code works as we pass pointer in `move_to_element()` which sets the pointer and `click()` uses that. 
```
ActionChains(driver).move_to_element(toClick, device=mouse).click()
```

Am I thinking right? Should `click()` be able to differentiate between `PointerInput` and `KeyInput`.
If yes, shall I mention this in the new issue which I am creating?